be/src/vec/columns/column_string.cpp

```diff
@@ -292,6 +292,7 @@ ColumnPtr ColumnString::replicate(const Offsets& replicate_offsets) const {
 
 void ColumnString::reserve(size_t n) {
     offsets.reserve(n);
+    chars.reserve(n * 5);
```

Comment:
why multi 5? add comment to explain, or define it as a variable.

Reply:
This is a mistake. In fact, it is `n`